### DISCUSSION TITLE
Server-hosted

### DISCUSSION THREAD

**Discussion by WashierPlayz** [vrcx-team/VRCX#848] [Ideas]
Is it possible to have a feature added so that VRCX can run on a self-hosted home server and send/receive info from the home-server so I can shut down my main PC once in a while to avoid some info being blanked out?

**Comment by Myrkie** [comment]
that would be an incredible amount of work and is completely out of the spec of this project as the code base isn't very readable in its current state.

**Comment by Nekromateion** [comment]
In the rewrite™ _Surely_
